github/event_types.go

```diff
@@ -7,16 +7,35 @@
 
 package github
 
+// Pusher represents the name and email address of a GitHub pusher.
+type Pusher struct {
```

Comment:
rather than create a new type, let's simply use a `User` instead.  Even though the GitHub API docs only document including the name and email fields, I have a feeling that this really is supposed to identify a GitHub user.